src/sidebar.c

```diff
@@ -857,7 +857,6 @@ static void on_openfiles_document_action(GtkMenuItem *menuitem, gpointer user_da
 			while (i >= 0 && gtk_tree_model_iter_nth_child(model, &child, &iter, i))
 			{
 				gtk_tree_model_get(model, &child, DOCUMENTS_DOCUMENT, &doc, -1);
-
```

Comment:
Should get rid of this change since it's unrelated and in another file.

Reply:
Ah, I thought I had deja-vu when writing above comment, it was in #2346 that this came from I guess.